test/lock_test.py

```diff
@@ -21,23 +21,34 @@
 import mock
 from helpers import unittest
 
+from tenacity import retry, retry_if_result, stop_after_attempt, wait_exponential
 import luigi
 import luigi.lock
 import luigi.notifications
 
 luigi.notifications.DEBUG = True
 
+pcmd_retry = retry(retry)
```

Comment:
Is this used for anything?

Reply:
Removed in [e8a957c](https://github.com/spotify/luigi/pull/3140/commits/e8a957cfa01f4f62f4df42359420e9f4815f437f)